### ISSUE TITLE
Contains and Contains Words seem to have a limit on the length of what they can match

### ISSUE THREAD

**Issue by StingyJack** [MicrosoftDocs/vsts-docs#6467]
If I have a longish URL (~150 chars) in the title or description of a workitem, I cant find that url by exact matching using `contains` or `contains words`.  If I trim my search criteria down to &lt; 100 chars the item starts returning in queries on the devops portal and when using the WIQL api

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6859c579-3383-e4b7-366e-69f49a8bd1b6
* Version Independent ID: 5b7af784-0698-b77a-342f-252986038e92
* Content: [Query by title, ID, or rich-text fields - Azure Boards](https://docs.microsoft.com/en-us/azure/devops/boards/queries/titles-ids-descriptions?view=azure-devops)
* Content Source: [docs/boards/queries/titles-ids-descriptions.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/boards/queries/titles-ids-descriptions.md)
* Product: **devops**
* Technology: **devops-agile**
* GitHub Login: @KathrynEE
* Microsoft Alias: **kaelli**

**Comment by msebolt** [comment]
@StingyJack Thank you for the feedback, assigning this to the author for review.

**Comment by KathrynEE** [comment]
Emailed product team for clarification.

**Comment by danhellem** [comment]
@StingyJack good catch. I believe 100 is the limit. We will work with docs and engineering to get the limits updated in the docs accordingly
